.github/workflows/slack-noti.yml

```
@@ -0,0 +1,16 @@
+name: Slack
+on: [pull_request]
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+    name: Slack Notification
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Fire Notification
+        uses: Lubycon/github-reviewer-slack-noti-action@v1.1.2
```

```suggestion
        uses: Lubycon/github-reviewer-slack-noti-action@v1.1.3
```

`v1.1.3` 에서 코드오너 많을 때 리뷰어 멘션이 제대로 가지 않는 이슈 수정되었씁니다!